symfit/core/objectives.py

```diff
@@ -322,7 +322,7 @@ def eval_hessian(self, ordered_parameters=[], **parameters):
                 p1 = hess_comp * ((y - f) / sigma**2)[np.newaxis, np.newaxis, ...]
                 # Outer product
                 p2 = np.einsum('i...,j...->ij...', jac_comp, jac_comp)
-                p2 /= sigma[np.newaxis, np.newaxis, ...]**2
+                p2 = p2 / sigma[np.newaxis, np.newaxis, ...]**2
```

Comment:
Why not `/=`?

Reply:
Broadcasting issues if the shape is changed during this operation.